jetty-server/src/main/config/etc/jetty-gzip.xml

```diff
@@ -13,11 +13,7 @@
     <Arg>
       <New id="GzipHandler" class="org.eclipse.jetty.server.handler.gzip.GzipHandler">
         <Set name="minGzipSize" property="jetty.gzip.minGzipSize"/>
-        <Set name="checkGzExists" property="jetty.gzip.checkGzExists"/>
-        <Set name="compressionLevel" property="jetty.gzip.compressionLevel"/>
         <Set name="inflateBufferSize" property="jetty.gzip.inflateBufferSize"/>
-        <Set name="inflaterPoolCapacity" property="jetty.gzip.inflaterPoolCapacity"/>
-        <Set name="deflaterPoolCapacity" property="jetty.gzip.deflaterPoolCapacity"/>
```

Comment:
I think this XML is out-of-date. It should be changed to allocate the `InflaterPool` and `DeflaterPool` and the relevant properties set on the pools (rather than on the handler as it is done here).

Reply:
Is it fine to take a all the default values for the `InflaterPool` and `DeflaterPool`?

Reply:
I think yes.

Note that I am going to open another issue.  We need a way to check that `property` attributes actually exist, else we can change the code and not notice the XML is wrong unless the property is actually set.     I think we should always validate that there is at least one matching setter, even if the property is not set.

Reply:
@sbordet  I have configured an `InflaterPool` and a `DeflaterPool`.  Note that I have had to change the names of the properties in the `.mod` file as they no longer make sense - I've put them in the `jetty-rewrite.xml ` as `deprecated`. I've used the default values that are present in the code.

Reply:
Sorry, working on too many things at once. They are of course in` jetty-gzip.xml`.